### DISCUSSION TITLE
no display

### DISCUSSION THREAD

**Discussion by kitmanten** [rustdesk/rustdesk#5807] [General]
### Bug Description

i rustdesk my mini machine whitch it with screen,at this time it is working.but when i rustdesk my only machine without it's screen,it hint **No Display**;
my mini machine is working on ubuntu 22.04;
so how can i fix it .

### How to Reproduce

rustdesk machine whitch without  screen ,and machine working on ubuntu 22.04.

### Expected Behavior

fix it

### Operating system(s) on local side and remote side

ucuntu22.04

### RustDesk Version(s) on local side and remote side

1.1.9

### Screenshots

![图片](https://github.com/rustdesk/rustdesk/assets/41369085/a3b3d026-a997-4f89-b2cc-ce8728da176e)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/wiki/Headless-Linux-Support
